src/widgets/PaperModal/PaperConfirmModal.js

```diff
@@ -25,25 +27,35 @@ export const PaperConfirmModal = ({
     </FlexView>
     <FlexRow justifyContent="space-evenly" alignItems="center" style={{ width: '100%' }} flex={1}>
       <PageButton onPress={onCancel} text={cancelText} textStyle={localStyles.cancelText} />
-      <PageButton
-        onPress={onConfirm}
-        text={confirmText}
-        textStyle={localStyles.confirmText}
-        style={localStyles.confirmButton}
-      />
+      {withOnePress ? (
+        <PageButtonWithOnePress
+          onPress={onConfirm}
+          text={confirmText}
+          textStyle={localStyles.confirmText}
+          style={localStyles.confirmButton}
+        />
+      ) : (
+        <PageButton
+          onPress={onConfirm}
+          text={confirmText}
+          textStyle={localStyles.confirmText}
+          style={localStyles.confirmButton}
+        />
+      )}
```

Comment:
I think the `PageButtonWithOnePress` is generally just a bit problematic? Could just use `PageButton` and solve the spam clicking problem through the callback

Using `debounce` or..

```
const singleInvoke = (fn)  => {
    let hasRun = false;
    return (...args) => {
        if (!hasRun) {
            hasRun = true;
            fn(...args);
        }
    };
};

```

Reply:
Yeah the OnePress thing was messing everything up 😆 . Thanks, I missed the callback on the `PageButton`, will give that a go

Reply:
Resolved the `PageButtonWithOnePress` stuff with c58ff4a and 0b02810